### PR TITLE
Add a README with the instructions to run the TCK

### PR DESCRIPTION
**Pull request by cescoffier**
@Emily-Jiang as discussed.

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Review by xstefank — APPROVED** [review]

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]
To make it consistent with others, can you add the following or something similar?

= Running the MicroProfile Reactive Streams Operators TCK

Any MicroProfile Reactive Streams Operators implementation must pass this test suite. The TCK uses `TestNG` and `Arquillian`.

== Hardware Requirements

All systems should meet the following recommended hardware requirements:

    * CPU running at 2.0 GHz or higher
    * 4 GB of RAM or more
    * Network access to the Internet

== Software Requirements

You can run MicroProfile Config TCK on platforms running the Solaris, Linux, Windows, and Mac OS with the following software installed:

    * Maven
    * JDK8+

**Comment by cescoffier** [comment]
We don't use Arquillian to pass the TCK. That's an option. The provided instructions is what we are using.

**Comment by cescoffier** [comment]
@Emily-Jiang Updated.

BTW, it's Apache Maven. Please fix it in MP Config.

File: tck/README.adoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
+    <artifactId>microprofile-reactive-streams-operators-tck</artifactId>
+    <version>${microprofile-reactive-streams.version}</version>
+    <scope>test</scope>
+</dependency>
+----
+
+The TCK used TestNG.
```
Comment (Emily-Jiang) [inline tck/README.adoc:41]:
```suggestion
The TCK uses TestNG.
```

**Review by Emily-Jiang — APPROVED** [review]
@cescoffier I made just one grammar change. Please accept it and merge your PR.

**Pull request merged**